### PR TITLE
My Profile Page

### PR DESCRIPTION
**Pull request by mykb23**

### CODE REVIEW

**Comment by mykb23** [comment]
https://res.cloudinary.com/mykb/image/upload/v1554316920/psu.png

File: index.php [hunk 1]
```diff
@@ -45,8 +45,9 @@
           case '/become-intern' :
           require __DIR__ . '/views/become-intern.php';
           break;
-        case '/intern/Akin' :
```
Comment (Victor-Ugwueze) [inline index.php]: why did you remove this line ?

Comment (mykb23) [reply index.php]: The line was causing conflict. that is why I removed it.
